includes/Assets.php

```diff
@@ -564,8 +567,6 @@ public function enqueue_front_scripts() {
             }
         }
 
-        wp_enqueue_script( 'dokan-login-form-popup' );
```

Comment:
Isn't this pop up needed?

Reply:
this popup is needed when the user is not logged in and is trying to follow/unfollow store, get store support, report abuse. I have deleted it from here and enqueued it when necessary